lago/providers/libvirt/vm.py

```diff
@@ -320,8 +319,7 @@ def export_disks(self, standalone, dst_dir, compress, *args, **kwargs):
                 standalone=standalone,
                 *args,
                 **kwargs
-            ) for disk in self.vm.disks
-            if disk.get('format') not in formats_to_exclude
+            ) for disk in self.vm.disks if not disk.get('skip-export')
```

Comment:
Please add it to the LagoInitFile docs, small issue but would make our life easier.